core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -99,6 +107,11 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
     // Update bookmark state locally.
     row.bookmarked = !wasPreviouslyBookmarked;
 
+    // Update bookmark in retrieved executions list and re-paginate
+    this.workflowExecutionsService.retrievedExecutionsList[
+      this.workflowExecutionsService.retrievedExecutionsList.indexOf(row)
+    ].bookmarked = !wasPreviouslyBookmarked;
+
```

Comment:
you need to reverse the change in line 120 incase of error to change the this.workflowExecutionsService.retrievedExecutionsList